### ISSUE TITLE
Monitoring and reminders for annotations on active projects

### ISSUE THREAD

**Issue by anngvu** [nf-osi/jobs#23]
See NFINT-622

Considered [synapsemonitor](https://github.com/Sage-Bionetworks/synapsemonitor), but this can't be used as is because:
- Email recipient(s) specified as CLI param and not dynamic/based on the actual data uploader, since intention was primarily for notification to a known DCC account. 
- Conditions on created/modified within the last *** time interval. However, what we really want to condition on is presence of annotations. The check will look at whether there are annotations yet on the entity, which could have been uploaded yesterday,  a week ago, or even a month ago. (Hmm, what is the average time it takes for files to be annotated?)
- Since triggers on the modified condition, and adding annotations is a modification, there would be an email asking for annotations after something has just been annotated...

Basically, this suggests custom handling.

This should: 
- Query Study table for **active** projects and select relevant studyFileViews
- Check file entities in studyFileView
    - If no files, don't do anything
    - If files present, check for core required annotations (e.g. resourceType)
- Make list subset of files without any annotations
- Get user id(s) who uploaded the data
- Send email with list

Frequency: Weekly (daily would be extremely annoying)
